core/src/commonMain/kotlin/info/laht/threekt/controls/OrbitControls.kt

```diff
@@ -98,12 +98,15 @@ class OrbitControls(
 
     private var state = State.NONE
 
+	private val defaultKeyListener = MyKeyListener()
```

Comment:
You use different indentation settings. Please reformat all files using official Kotlin style.